vars/main.yml

```diff
@@ -23,6 +23,22 @@ core:
   upf:
     ip_prefix: "192.168.252.0/24"
     iface: "access"
+    # when using 'dpdk' mode please make sure atleast two VF devices are
+    # created out of 'data_iface'
+    mode: af_packet          # Options: af_packet or dpdk
+    # set to true if mode is dpdk. This will enable the required resources
+    # like hugepage, sriov, etc...
+    enable_resources: false
+    # configure the PCI address of VF's which created out of 'data_iface'.
+    # It will be used for N3, N6 interfaces. Configure these values only if
```

Comment:
```suggestion
    # It will be used for N3 and N6 interfaces. Configure these values only if
```

Reply:
code removed, so Not Applicable anymore.